doc/api/next_api_changes/behavior/25775-HZ.rst

```diff
@@ -0,0 +1,42 @@
+Support customizing antialiasing for text and annotation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+
+``matplotlib.pyplot.annotate()`` and ``matplotlib.pyplot.text()`` now support parameter ``antialiased``.
+When ``antialiased`` is set to ``True``, antialiasing will be applied to the text.
+When ``antialiased`` is set to ``False``, antialiasing will be applied to the text.
+When ``antialiased`` is not specified, it will use the value from ``rcParams['text.antialiased']``.
+
+Examples:
+
+.. code-block::
+
+    mpl.text.Text(.5, .5, "foo\nbar", antialiased=True)
+    plt.text(0.5, 0.5, '6 inches x 2 inches', antialiased=True)
+    ax.annotate('local max', xy=(2, 1), xytext=(3, 1.5), antialiased=False)
+
+If the text contains math expression, then anaialiasing will be applied by ``rcParams['text.antialiased']`` and ``antialiased`` will have no effect
+This applies to the whole text.
+Examples:
+
+.. code-block::
+
+    plt.text(0.5, 0.25, r"$I'm \sqrt{x}$", antialiased=False)
+
+Also note that antialiasing for coordinate axes will be set with ``rcParams['text.antialiased']`` when they are created and cannot be changed afterwards.
```

Comment:
```suggestion
Also note that antialiasing for tick labeles will be set with ``rcParams['text.antialiased']`` when they are created and cannot be changed afterwards.
```

Reply:
Axes vs axis have very specific meaning in Matplotlib (`matplotlib.axes.Axes` vs `matplotlib.axis.Axis`).  I think tick labels (our name for the text on the ticks), see https://matplotlib.org/stable/gallery/showcase/anatomy.html for the very particular names we use for things.